packages/components/addon-main.cjs

```diff
@@ -4,4 +4,13 @@
  */
 
 const { addonV1Shim } = require('@embroider/addon-shim');
-module.exports = addonV1Shim(__dirname);
+const flightIconSprite = require('@hashicorp/flight-icons/svg-sprite/svg-sprite-module');
```

Comment:
I see the same changes as in https://github.com/hashicorp/design-system/pull/2208
Please base this second PR on the first one (#2208) not on the feature branch

(plus, make sure the tests are passing)

Let me know once done, so I can continue the review

Reply:
All set now. Thanks for reviewing.